src/services/DiscourseService.ts

```diff
@@ -49,7 +49,7 @@ export class DiscourseService {
     }
 
     return {
-      category: DISCOURSE_CATEGORY ? Number(DISCOURSE_CATEGORY) : undefined,
+      category: DiscourseService.DISCOURSE_CATEGORY ? Number(DiscourseService.DISCOURSE_CATEGORY) : undefined,
```

Comment:
If `DISCOURSE_CATEGORY` is a required env for the server to run then this condition is not necessary.

```
return {
  category: Number(DISCOURSE_CATEGORY)
  ...
}
```

But maybe typescript throws an error here, not sure what requiredEnv returns.

Reply:
🤔 
It should always return a string, but if it's an empty string, it would be turned into `0`
I don't know if requiredEnv throws if you have an empty string, but I'd rather not change this right now :)